python-package/README.md

```diff
@@ -24,8 +24,8 @@ Contribute it to the _nichtparasoup_ project or write it as an independent plugi
 ---
 
 
-At our hackspace [k4cg](https://k4cg.org) we use it since years now.  
-It turns out to be a very non-invasive way of entertaining a crowd of nerds 
+At our hackspace [k4cg](https://k4cg.org) we've been use it for years now.  
```

Comment:
the "use" is not done, yet. it is still in progress.
so i  guess this is not past perfect "been using"  ...